integration/hostapp/mobynit_test.go

```diff
@@ -0,0 +1,53 @@
+package hostapp
+
+import (
+	"context"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/network"
+	"github.com/docker/docker/cmd/mobynit/hostapp"
+	"github.com/docker/docker/internal/test/daemon"
+
+	"gotest.tools/assert"
+	"gotest.tools/skip"
+)
+
+func TestMobynitMountContainer(t *testing.T) {
+	skip.If(t, testEnv.DaemonInfo.OSType != "linux")
+	skip.If(t, testEnv.IsRemoteDaemon, "cannot start daemon on remote test run")
+	defer setupTest(t)()
+
+	d := daemon.New(t)
+	d.StartWithBusybox(t)
+	defer d.Stop(t)
+
+	client, err := d.NewClient()
```

Comment:
@roman-mazur are you going to update this to the same one as in #213

Reply:
done